ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,25 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Catholic University of America (CUA)
```

Comment:
```suggestion
 - The Catholic University of America (CUA)
```
I checked online and the name officially starts with 'The'. Technically it was the same thing at UTD. CUA should be placed at the end.

Reply:
"The" is ignored when alphabetizing.

Reply:
Depends upon the standard.